### PR TITLE
Create signUp.html

### PR DESCRIPTION
**Pull request by nik-6947**
I have made a front end of signup page.

### CODE REVIEW

**Comment by hkasera** [comment]
Please add screenshots for the same on how it looks on mobile as well as desktop.

File: signUp.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<html>
+<head>
+<meta charset="utf-8">
+<title>Sign Up</title>
+<style type="text/css">
+body {
+	background: url("http://wp.givingtuesday.ca/app/uploads/2014/11/GivingTuesday_BlogHeader.jpg") no-repeat center center fixed;
```
Comment (hkasera) [inline signUp.html]: * Why not a separate CSS file?
* Better to download the image and serve as a local resource than serving over the internet.

File: signUp.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<html>
```
Comment (hkasera) [inline signUp.html]: Always prettify your file for better readability. There are lots of tools online!

File: signUp.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+<head>
+<meta charset="utf-8">
```
Comment (hkasera) [inline signUp.html]: Check out other meta tags that you may need. Also, please close the tag.

**Comment by nik-6947** [comment]
hey, if we want image to serve as a local source, i need to upload that with html and css files, but i think we are not allowed to upload files.
Can you pls guide us in this?
and i am soon uploading the seperated html and css files.
Thank u.

**Comment by hkasera** [comment]
@nik-6947 Can you explain what do you mean by not allowed to upload files?

**Comment by nik-6947** [comment]
hey , i m sry that was my git's setting problem. I was getting this error.
screenshot from 2016-04-08 15 28 12

My partner nishtha3006 has uploaded all the files.

**Comment by hkasera** [comment]
Review each other's change and then only send a PR. Send a new pull request by thoroughly testing your changes. If you have sent a PR and I have asked to fix some issues, edit the same PR and don't create new ones. For now, I am closing this PR also. Create a fresh PR properly by testing your changes and commits.

**Pull request closed without merging**